pages/assets/js/src/profile-pages/pg-dashboard.ts

```diff
@@ -37,6 +37,18 @@ export class PgDashboard extends OpenElement {
       });
   }
 
+  async connectedCallback() {
+    super.connectedCallback();
+
+    //if no location is saved on the user, get it from the IP and save it to the user
+    if ( !this.user.location_hash.length ) {
+      await this.getLocationFromIP();
+    }
+    //link anonymous prayers to the user
+    //@todo save the user id locacally so this happens automatically
```

Comment:
as in, even if they are logged out, the user id is in localstorage, so prayers get linked to their pre-logged in user? (thinking re the @todo point)